glean-core/python/glean/_ffi.py

```diff
@@ -31,7 +35,40 @@ def get_shared_object_filename() -> str:  # pragma: no cover
 
 
 lib = ffi.dlopen(str(Path(__file__).parent / get_shared_object_filename()))
-lib.glean_enable_logging()
+
+
+# TODO: Just for testing, let's show everything
+logging.basicConfig(level=logging.DEBUG)
+
+
+def setup_logging():
+    r, w = os.pipe()
+    lib.glean_enable_logging(w)
+
+    reader = os.fdopen(r)
+
+    name = "glean.rust_core"
+    log = logging.getLogger(name)
+    level_map = {
+        "CRITICAL": logging.CRITICAL,
+        "ERROR": logging.ERROR,
+        "WARNING": logging.WARNING,
+        "INFO": logging.INFO,
+        "DEBUG": logging.DEBUG,
+    }
+
+    def log_handler():
+        while True:
+            data = reader.readline().rstrip()
+            json_content = json.loads(data)
+            log.log(level_map.get(json_content["level"], 0), json_content["message"])
+
+    logging_thread = threading.Thread(target=log_handler)
```

Comment:
I have a slight dislike that this requires a thread, but I guess with this design it's the only way we can ensure to continously getting all data.